### PR TITLE
Abstracts with ILOs

### PR DESCRIPTION
**Pull request by dbosk**
@lenfra check if you agree with the ILOs for the assignments (the memos and seminars).

### CODE REVIEW

File: recmgmt/abstract.tex [hunk 1]
```diff
@@ -14,5 +14,10 @@
 
 The lecture will be an introduction to archives and information science, basic 
 concepts, processes, business process analysis and information mapping.
-It covers material from primarily \citetitle{infokartl}~\cite{infokartl} and 
-the standard ISO 30300:2011~\cite{ISO30300:2011}.
+More concretely, after this session you should be able to
+\begin{itemize}
+  \item \dots
```
Comment (dbosk) [inline recmgmt/abstract.tex]: Fix or remove.

Comment (dbosk) [reply recmgmt/abstract.tex]: Fixed.

**Comment by dbosk** [comment]
Now I've also added an abstract with ILOs for the project.

**Comment by lenfra** [comment]
These were made for the previous materials, need to update for latest version. I will fix this.

**Comment by dbosk** [comment]
Excellent, thanks!

**Pull request closed without merging**